package.json

```diff
@@ -16,14 +16,16 @@
     "url": "https://github.com/webcomponents/custom-elements/issues"
   },
   "scripts": {
+    "install": "$(npm bin)/bower install",
```

Comment:
This causes things to break for me if I `npm install @webcomponents/custom-elements`. Since bower is a devDependency, it is not installed but the `install` script still tries to use it.

What is the purpose of this `install` script? When somebody is `npm install`ing this project, they don't need the bower_components, do they?